<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signalwindow</class>
 <widget class="QDialog" name="signalwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="signalwindow_pidlabel">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>pid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="signalwindow_pid">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="signalwindow_Namelabel">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="signalwindow_Name">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="signalwindow_signalComboBox">
     <property name="placeholderText">
      <string>Сигнал</string>
     </property>
     <item>
      <property name="text">
       <string>1 SIGHUP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 SIGINT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 SIGQUIT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 SIGILL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 SIGTRAP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 SIGABRT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7 SIGBUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 SIGFPE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9 SIGKILL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 SIGUSR1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11 SIGSEGV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 SIGUSR2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13 SIGPIPE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14 SIGALRM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 SIGTERM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17 SIGCHLD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18 SIGCONT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19 SIGSTOP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20 SIGTSTP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21 SIGTTIN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22 SIGTTOU</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>signalwindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>signalwindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
